src/manage_nsfs/manage_nsfs_help_utils.js

```diff
@@ -458,6 +477,88 @@ Usage:
 `;
 
 
+const CONNECTION_FLAGS_ADD = `
+Help:
+
+    Use this CLI to add a connection.
+
+Usage:
+
+    connection add [flags]
+
+Flags:
+
+    --name <string>                                                         Set the name for the connection
+    --agent_request_object <object>                                         Value of agent request objects, used for http(s) connection, as defined by nodejs http(s) agent options
+    --request_options_object <object>                                       Value of http(s) request option, as defined by nodejs http(s) request option.
+    --notification_protocol <string>                                        One of http, https, kafka.
+    --from_file <string>                                  (optional)        Use details from the JSON file, there is no need to mention all the properties individually in the CLI
+
+`;
+
+const CONNECTION_FLAGS_UPDATE = `
+Help:
+
+    Use this CLI to update a connection.
+
+Usage:
+
+    connection update [flags]
+
+Flags:
+
+    --name <string>                                                         The name of the connection to update.
+    --key <string>                                                          Name of field to update
+    --value <string>                                                        Value of the field to update
+    --remove_key <string>                                                   Removes a key from the connection.
+`;
+
+const CONNECTION_FLAGS_DELETE = `
+Help:
+
+    Use this CLI to delete a connection.
+
+Usage:
+
+    connection delete [flags]
+
+Flags:
+
+    --name <string>                                                         The name of the connection to delete.
+
+`;
+
+const CONNECTION_FLAGS_STATUS = `
+Help:
+
+    Use this CLI to get connection status.
+
+Usage:
+
+    connection status [flags]
+
+Flags:
+
+    --name <string>                                                         The name of the connection.
+    --decrypt <boolean>                                                     Wether to decrypt the auth field of the request.
+
+`;
+
+const CONNECTION_FLAGS_LIST = `
+Help:
+
+    Use this CLI to list connections.
+
+Usage:
+
+    connection list [flags]
```

Comment:
When we list accounts/buckets we also have the option `--wide` to get more information (and not just the name). You can consider adding it (not necessarily in this PR).

Reply:
I think it's too early for this. Let's see if anyone uses list at all.